### PR TITLE
Fixed a bug in the spawnBalls method that lead to balls of negative s…

### PR DESCRIPTION
**Pull request by skoppers**
…ize be created

<bug>

### CODE REVIEW

**Comment by jorisoudejans** [comment]
Isn't there an easier way to do the random generator? Without Math.abs?

**Comment by hlycklama** [comment]
I'd like to see some tests first

**Comment by skoppers** [comment]
You could use System.CurrentTimeMillis % 5. The problem is, is that it then is unable to give a seed, thus making testing the method impossible.

Test are on the way.

File: src/main/java/doob/model/Level.java [hunk 1]
```diff
@@ -184,22 +184,9 @@ private void updateSurvival() {
...
-                break;
-            case 3: spawnSameSizeBalls(2, 64);
-                break;
-            case 4: spawnSameSizeBalls(1, 128);
-                break;
-            default:
-                break;
-        }*/
-        spawnSameSizeBalls(6 - i, (int) (8 * Math.pow(2, i)));
+        int i = Math.abs(generator.nextInt() % 4);
```
Comment (jorisoudejans) [inline src/main/java/doob/model/Level.java]: I mean this line. Isn't there a better way to use a random generator than to use Math.abs()?

Comment (skoppers) [inline src/main/java/doob/model/Level.java]: Yes it may be possible, but it wouldn't necessarily be less complex.

**Comment by jorisoudejans** [comment]
Test methods should only have one or two asserts. You can better split the methods into multiple smaller ones.

**Comment by jorisoudejans** [comment]
Yes, all looks OK, can you resolve the merge conflicts?

**Pull request merged**